File: 1720-crawler-log-folder/1720-crawler-log-folder.go
func minOperations(logs []string) int {
    var stack[] string
    for _ , log :=  range logs{
        if log == "./"{
            continue 
        }else if len(stack) > 0 && log =="../"{
            stack = stack[:len(stack)-1]
        }else if log != "../"{
            stack = append(stack , log)
        }
    }
    return len(stack)
}

File: 1720-crawler-log-folder/1720-crawler-log-folder.java
class Solution {
    public int minOperations(String[] logs) {
        
        Stack<String> stack = new Stack<>();
      
        for(int i = 0 ; i < logs.length; i++){
            if(logs[i].equals("./")){
                continue;
            }else if (!stack.isEmpty() && logs[i].equals("../")){
                stack.pop();
            }else if (!logs[i].equals("../")) {
                stack.push(logs[i]);
            }
        }
      
        return stack.size() ;
    }
}